ros_gz_sim_demos/README.md

```diff
@@ -14,17 +14,12 @@ There's a convenient launch file, try for example:
 
 Publishes fluid pressure readings.
 
-    ros2 launch ros_gz_sim_demos air_pressure.launch.py
+    ros2 launch ros_gz_sim_demos air_pressure.launch
```

Comment:
```suggestion
    ros2 launch ros_gz_sim_demos air_pressure.launch.xml
```

Reply:
Added in [643a69b](https://github.com/gazebosim/ros_gz/pull/632/commits/643a69b9ca2f2630e103d082f90f276f5274c8f0).